knowledge/sports/athletes/caitlin_clark/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: bob longmore
+domain: anatomy
+seed_examples:
+- answer: Iowa
+  question: Where did Caitlin Clark go to college?
+- answer: Dowling Catholic High School
+  question: What high school did Caitlin Clark attend?
+- answer: two
+  question: How many times was Caitlin Clark voted AP Player of the Year?
+- answer: zero
+  question: How many national championships did Caitlin Clark win?
+- answer: Indiana Fever
+  question: What WNBA team drafted Caitlin Clark?
+task_description: ' '
+document:
+ repo: https://github.com/juliadenham/Summit_knowledge
+ commit: 481b735225e2714c4e9d36da6d7b941d9a36bd86
+ patterns:
+  - wisdom_tooth.md
```

Comment:
Strange to call the file "wisdom_tooth.md" if you are making questions about Caitlin Clark.